tests/Doctrine/Tests/ORM/Functional/Ticket/DDC1404Test.php

```diff
@@ -40,12 +40,10 @@ protected function setUp(): void
 
     public function testTicket(): void
     {
-        $this->expectDeprecationWithIdentifier('https://github.com/doctrine/orm/issues/8592');
-
         $repository  = $this->_em->getRepository(DDC1404ChildEntity::class);
-        $queryAll    = $repository->createNamedQuery('all');
-        $queryFirst  = $repository->createNamedQuery('first');
-        $querySecond = $repository->createNamedQuery('second');
+        $queryAll    = $repository->createQueryBuilder('p')->getQuery();
+        $queryFirst  = $repository->createQueryBuilder('p')->where('p.id = 1')->getQuery();
+        $querySecond  = $repository->createQueryBuilder('p')->where('p.id = 2')->getQuery();
```

Comment:
You can remove this file entirely, as it was introduced in 8af0f9d0711a7dcba8364e86c29cc3cc6e426010

The title of that commit says "added support for Inherited Named Queries" so that test is meant to test named query.